frontend/src/components/review/StatusBadge.tsx

```diff
@@ -124,6 +124,14 @@ const StatusBadge = ({
         >
           Pending
         </MenuItem>
+        <MenuItem
+          icon={<Icon as={MdTimer} height={6} width={6} />}
```

Comment:
![image](https://user-images.githubusercontent.com/15113249/147178581-46d89e6a-0f01-4d55-a505-be664ded9b62.png)
mind using this one? kinda confusing that pending & action use the same ones
(not designer approved but imo this makes sense)